py/server/deephaven/table.py

```diff
@@ -319,6 +320,34 @@ def meta_table(self) -> Table:
     def j_object(self) -> jpy.JType:
         return self.j_table
 
+    def get_table_attributes(self) -> Dict[str, Any]:
```

Comment:
This method seems to be studdering by having `table` in it.  Also, `get` is very java like.  Should it just be `attributes`?